codalab/model/bundle_model.py

```diff
@@ -643,30 +656,15 @@ def batch_get_bundles(self, **kwargs):
         ]
         return bundles
 
-    def set_waiting_for_worker_startup_bundle(self, bundle, job_handle):
```

Comment:
This got renamed, had its comment updated and moved through the file. No changes to business logic.